src/Buttons/Button.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  useContext,
+  ReactNode,
+  MouseEventHandler,
+} from 'react';
+import { VARIANT } from 'Theme/MarkOneTheme';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from '../Theme';
+
+export interface ButtonProps {
+  /** Text or components to be displayed on the button */
+  children?: ReactNode;
+  /** Function to call on click event */
+  onClick: MouseEventHandler;
+  /** If true, button won't fire */
+  disabled?: boolean;
+  /** Allows you to pass in style properties for the element */
+  theme?: object;
+  /** Allows you to pass in a variant property from the VARIANT enum */
+  variant: VARIANT;
+}
+
+const StyledButton = styled.button<ButtonProps>`
+  background: ${({ theme, variant }): string => (theme.color.background[variant].medium)};
+  border-color: ${({ theme, variant }): string => (theme.color.background[variant].medium)};
```

Comment:
@jonseitz I'm a bit confused about what color to use for the border color, because originally I used the gray border for all my buttons, and then Chris asked me to change the border color to a color slightly darker than the button color (but if the color is `medium` and `dark` on hover, then we would need a `darker` option for the border on hover). Then in Vittorio's latest screenshot (below) he has the same border color as the button except for the basic button, but he also noted that the color contrast between the button text and color is enough that we don't need a darker border. Should I change the border to `theme.border.light` for now and then we can discuss in the course planning meeting? 
![image](https://user-images.githubusercontent.com/29589689/75266116-df99f380-57bf-11ea-8dc2-d9a020437337.png)

Reply:
Removed borders in response to our discussion through commit 4b66bd2